components/MatterInterface/DoneMatterEndpoint.cpp

```diff
@@ -7,17 +7,17 @@ static const char *TAG = "DoneMatterEndpoint";
 
 namespace esp_matter {
 using namespace cluster;
-
```

Comment:
add comment for this source

Reply:
I just wanted to let you know that there is no additional explanation. These codes are the continuation of the development of esp_matter::cluster.